INFO     Logger:page.py:122 Page.wait_until_stable() - Page wait until driver stable
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8da50>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8d990>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8d810>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
INFO     Logger:page.py:122 Page.wait_until_stable() - Page wait until driver stable
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8f670>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8f790>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
WARNING  urllib3.connectionpool:connectionpool.py:824 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f61b6a8c8b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /wd/hub/session
INFO     Logger:page.py:122 Page.wait_until_stable() - Page wait until driver stable
INFO     Logger:page.py:216 Page.maximize_window() - Maximize browser window
INFO     Logger:page.py:55 User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
INFO     Logger:page.py:107 Page.visit() - Get URL: 'https://next.4zida.rs'
INFO     Logger:page.py:110 Client window width: 393, window height: 873
INFO     Logger:page.py:259 The title of the page is 'Pretraga Nekretnina za Prodaju i Izdavanje - 4zida'
INFO     Logger:page.py:209 Page.screenshot() - Save screenshot to: 'screenshots/test_browser_title2024-02-29T092942.png'
INFO     Logger:page.py:201 Page.quit() - Quit page and close all windows from the browser session